grpc/util.go

```diff
@@ -41,10 +41,15 @@ func ClientSetup(c *cmd.GRPCClientConfig) (*grpc.ClientConn, error) {
 	if err != nil {
 		return nil, err
 	}
+	if stats == nil {
```

Comment:
Is there any place we actually call this with nil stats? Wouldn't it be better to error out?